src/Microsoft.CodeAnalysis.NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/xlf/MicrosoftCodeQualityAnalyzersResources.cs.xlf

```diff
@@ -2,9 +2,59 @@
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="cs" original="../MicrosoftCodeQualityAnalyzersResources.resx">
     <body>
+      <trans-unit id="AbstractTypesShouldNotHaveConstructorsDescription">
```

Comment:
😕 What happened here?

Reply:
I believe moving the resource files somehow caused the xlf files to get regenerated. I am going to investigate if I can avoid this change.

Reply:
Done, reverted.